src/main/kotlin/no/nav/helse/prosessering/v1/PdfV1Generator.kt

```diff
@@ -195,10 +198,6 @@ private fun List<Organisasjon>.erAktuelleArbeidsgivere() = any { it.skalJobbePro
 
 private fun String.formaterId() = if (length == 11) "${this.substring(0,6)} ${this.substring(6)}" else this
```

Comment:
Det var visst bare til irritasjon at det ble formatert, så endre dette til å bare returnere String'en as is

Reply:
Glemte søkers fødselsnummer. Fiksa nå :)